mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/impl/RequestState.scala

```diff
@@ -336,8 +336,8 @@ import scala.util.{Failure, Success}
    * In case some brokers treat 0 as no packet id, we set our min to 1
    * e.g. https://renesasrulz.com/synergy/synergy_tech_notes/f/technical-bulletin-board-notification-postings/8998/mqtt-client-packet-identifier-is-0-by-default-which-causes-azure-iot-hub-to-reset-connection
    */
-  private val MinPacketId = PacketId(1)
-  private val MaxPacketId = PacketId(0xffff)
+  val MinPacketId = PacketId(1)
+  val MaxPacketId = PacketId(0xffff)
```

Comment:
These are enclosed in a `private[streaming]` already. I am using them in the tests, hence removing the private here.